### ISSUE TITLE
Light and Dark themes

### ISSUE THREAD

**Issue by sanskar009** [ram-on/SkyTube#799]
Hi , I would like to enhance the Android app by adding "**Separate light and dark themes**"

**Comment by lakshaysr4** [comment]
@gzsombor @ram-on Can I work on this?

**Comment by gzsombor** [comment]
Sure, any contribution is welcomed! :+1:

**Comment by sanskar009** [comment]
Hi @gzsombor  & @ram-on , me and @lakshaysr4 are a part of a team consisting of college students that will be working on this issue for our Integrated Software Development course under Team Software Process (TSP) guidelines. On that note, can you please share your acceptance criteria with us?

**Comment by gzsombor** [comment]
I don't know exactly, what @sanskar009 originally thought, but I guess, a switch is needed among the preferences to control and save the selected theme, and changing that value should trigger refresh of the UI with the newly picked theme.

**Comment by sanskar009** [comment]
Hi @gzsombor  & @ram-on , me and @lakshaysr4  are working on this issue so please add the assigned tag on issue .

**Comment by gzsombor** [comment]
After looking through the style handling in the app, I have two suggestions:
* it would be good, if the new themes could be based on **Theme.MaterialComponents.Light** or **Theme.MaterialComponents** instead of the current **Theme.AppCompat.Light.DarkActionBar**, so we could use material components too, like com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton 
* in the layout xmls, I fear, there are two many theme specific overrides, like  android:theme, android:style, android:textAppearance, it would be nice, if we could remove these ones - otherwise I guess, we need separate overrides for light and black themes

**Comment by sanskar009** [comment]
@gzsombor yeah we are working in the same manner . The themes will be based on **Theme.MaterialComponents.Light** .

**Comment by R8YsDcYwuy3** [comment]
@sanskar009 Is this still being worked on?

**Comment by sanskar009** [comment]
Sorry but no .
